spec/openapi.yaml

```
@@ -3950,6 +3997,297 @@ components:
+                      type: string
+                      nullable: true
+                      enum:
+                        - F
+                        - M
+                invoiceAddress:
+                  description: Invoice address of the customer
+                  type: object
+                  properties:
+                    id:
+                      description: Address identifier
+                      type: string
+                      format: uuid
+                    firstName:
+                      description: First name
+                      type: string
+                    lastName:
+                      description: Last name
+                      type: string
+                    company:
+                      description: Company name
+                      type: string
+                      nullable: true
+                    street:
+                      properties:
+                        name:
+                          description: Street name
+                          type: string
+                        number:
+                          description: Street number
+                          type: string
+                        suffix:
+                          description: Street number suffix
+                          type: string
+                          nullable: true
+                      nullable: true
+                    zipCode:
+                      description: Zip code
+                      type: string
+                    city:
+                      description: City
+                      type: string
+                    countryCode:
+                      description: ISO 3166-1 alpha-2 country code
+                      type: string
+                      pattern: '^[A-Z]{2}$'
+                    phone:
+                      description: Phone number
+                      type: string
+                      nullable: true
+                    gender:
+                      description: Gender
+                      type: string
+                      nullable: true
+                      enum:
+                        - F
+                        - M
+        properties:
+          description: Payment properties
+          type: object
+          additionalProperties: {}
+    PaymentMutation:
+      example:
+        id: "1"
+        type: "authorize"
+        reference: null
+        price: "499"
+        description: null
+        createdAt: "2021-06-29T14:41:26+02:00"
+        expiresAt: null
+      properties:
+        id:
+          description: Mutation ID
+          type: number
+        type:
+          description: Mutation type
+          type: string
+          enum:
+            - "authorize"
+            - "capture"
+            - "failed"
+            - "refund"
+            - "void"
+        reference:
+          description: Mutation reference
+          type: string
+          nullable: true
+        price:
+          description: Price (in cents)
```

```suggestion
          description: Price in minor units
```
(for consistency with other fields)